### PR TITLE
style: add animation to ProgressCircular component

### PR DESCRIPTION
**Pull request by wildergd**
fix: #953 

## Changes proposed in this PR:
 - add animation to progress circular component

[ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@nexxtway/react-rainbow

### CODE REVIEW

File: src/components/ProgressCircular/progressRing.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 import PropTypes from 'prop-types';
 
 export default function ProgressRing({ percent }) {
-    const isLongArc = percent > 50 ? 1 : 0;
-    const arcX = Math.cos((2 * Math.PI * percent) / 100);
-    const arcY = Math.sin((2 * Math.PI * percent) / 100);
+    const [percentValue, setPercentValue] = useState(0);
+    useEffect(() => {
+        setPercentValue(percent);
+    }, [percent]);
```
Comment (LeandroTorresSicilia) [inline src/components/ProgressCircular/progressRing.js]: here if you only call setPercentValue when percent change then you can use percent directly, so there is not need to use useState and useEffect

File: src/components/ProgressCircular/readme.md [hunk 2]
```diff
@@ -13,12 +13,12 @@
 
         increment() {
             const { value } = this.state;
-            if (value === 100) {
+            if (value == 100) {
```
Comment (LeandroTorresSicilia) [inline src/components/ProgressCircular/readme.md]: why change to ==?

File: src/components/ProgressCircular/readme.md [hunk 3]
```diff
@@ -1,40 +1,96 @@
...
-        render() {
-            const { value } = this.state;
-            return (
-                <div className="rainbow-align-content_center rainbow-m-around_xx-large rainbow-flex_column">
-                    <ProgressCircular value={value} />
-                </div>
-            );
-        }
+        return (
+            <Card title={
```
Comment (TahimiLeonBravo) [inline src/components/ProgressCircular/readme.md]: Please review the examples markup, we should not add needlessly wraps.

**Comment by TahimiLeonBravo** [comment]
Review the examples in mobile, we need to make responsive the examples
View the images attached 👇
Screen Shot 2019-09-18 at 4 46 02 PM
Screen Shot 2019-09-18 at 8 58 39 PM

**Pull request merged**